docs/en/Concept.md

```diff
@@ -0,0 +1,41 @@
+# Basic Concept                                
+##
+## 1 Message Model
+
+The RocketMQ message model is mainly composed of a Producer, a Broker and a Consumer.The Producer is responsible for producing messages, the Consumer is responsible for consuming messages, and the Broker is responsible for storing messages.     
+A Broker corresponds to one server during actual deployment, and each Broker can store messages from multiple topics, and messages from each Topic can be stored in a different Broker by sharding strategy.     
+A Message Queue is used to store the physical addresses of messages, and the Message addresses in each Topic are stored in multiple Message queues.The Consumer Group consists of multiple Consumer instances.    
```

Comment:
1. "A Message Queue" => "A" shouldn't be used since Message Queue is a proper noun.
2. Seems that "physical offset" is more precise than "physical offset", though "物理地址" is used in Chinese doc.